readme.md

```diff
@@ -45,41 +44,41 @@ giving a general but no inevitably defined idea of where we're going next
-If you are a Sauerbraten developer worrying about us changing stuff and we're "totally going in the wrong direction", check out the posts in our blog called
+If you are a developer familiar with Sauerbraten worrying about us changing stuff and we're "totally going in the wrong direction", check out the posts in our blog called
 "Design Decisions" ([Design-Decisions Part1](https://inexor.org/blog/2015/04-26-alpha1-design-decisions), [Design-Decisions Part2](https://pad.inexor.org/p/Alpha3_Design_Decisions)) giving you a vague idea of our reasoning for some points.  
-(You'll probably find more info on the dedicated wiki-sites, in the readme of the particular module and for specific stuff in the [documentation](http://docs.inexor.org) )
+You'll probably find more info on the dedicated wiki-sites, in the readme of the particular module and for specific stuff in the [documentation](https://docs.inexor.org).
+
 
+A little note on that is that a lot of our decisions are consequences of other ones (e.g. working as a team had influences on the whole build structure, that we use CMake, that we use Git and hence had to split the data and code repository and more ...  it's all chained).
+Consequently, our approach in developing Inexor is probably the most profound one in the Cube engine world.
 
-A little note on that is that a lot of our decisions are consequences of other ones (e.g. working as a team had influences on the whole build structure, that we use cmake, that we use git and hence had to split the data and code repository and more ..  it's all chained).
-Consequently, our approach in developing Inexor is probably the most profound one in the Cube-world.
 
+## Build
 
-# First Steps
+Good tutorials on the whole process needed to build Inexor can be found in its [wiki page](https://github.com/inexor-game/code/wiki/Build), which you should read beforehand for saving you some nerves.
 
-You have already accomplished the first step by reading this readme. Congratz!
 
-The second one is only slightly harder by joining us in IRC (again see the wiki site for that: [Contact](https://github.com/inexor-game/code/wiki/Development-Environment)) and idling/participating in our chat.
+## Join us
 
-## Build
+You have already accomplished the first step by reading this readme. Congratz!
 
-Mainly it is:
+The second one is only slightly harder by joining us in IRC (again see the wiki site for that: [Contact](https://github.com/inexor-game/code/wiki/Contact)) and idling/participating in our chat.
 
-1. fetching our needed build dependencies (cmake, git)
-2. downloading the code and data repository using git
-3. creating your specific build-files (makefile/Visual Studio-solution/XCode project/..) using CMake
-4. and lastly, building Inexor
+To contribute to the project and merge your changes into the master branch you need to convince enough members that the changes are an improvement.
```

Comment:
Maybe reformulate this to be less negative? "Convince enough members" sounds like a big job.
"it wont be an issue" sounds like there have been issues before.

We preferably should create a wiki page ("How do I dive into Inexor?") which could give general strategies for diving into our development process with different entry points.
(Best would be in a flow graph like manner: [Are you already familiar with the Cube Codebase?] -> [Yes/No] -> [Do you already have programming skills]..
People love graphs.

We furthermore need to convince people to not being afraid to "sound stupid" and "not being good enough".